### PR TITLE
fix(docs): add no-invalid-icu eslint rule

### PR DESCRIPTION
**Pull request by cungminh2710**
This PR includes `no-invalid-icu` eslint rule in the docs

### CODE REVIEW

File: website/docs/tooling/linter.md [hunk 1]
```diff
@@ -646,3 +646,11 @@ Default complexity limit is 20 (using [Smartling as a reference](https://help.sm
   }
 }
 ```
+
+### `no-invalid-icu`
+
+This validates the ICU syntax.
+
+####
```
Comment (pyrocat101) [inline website/docs/tooling/linter.md]: nit: did you meant to write `#### Why`?

Comment (cungminh2710) [inline website/docs/tooling/linter.md]:
```suggestion
#### Why
```

Comment (cungminh2710) [reply website/docs/tooling/linter.md]: yeah oops 😆

**Review by pyrocat101 — APPROVED** [review]

**Comment by pyrocat101** [comment]
Thanks for the contribution!

**Pull request merged**